plugins/flatpak/gs-plugin-flatpak.c

```diff
@@ -274,6 +274,9 @@ gs_plugin_flatpak_get_handler (GsPlugin *plugin, GsApp *app)
 	GsPluginData *priv = gs_plugin_get_data (plugin);
 	const gchar *object_id;
 
+	if (gs_app_has_quirk (app, AS_APP_QUIRK_IS_PROXY))
+		goto select_by_scope;
```

Comment:
:scream_cat:

Reply:
https://github.com/endlessm/gnome-software/commit/ec25ff8c5001a0d9923c4b1a7ab9f906504d2338#diff-1023bde5962b1305d7ead31178586dd8R281

Reply:
Alternatively we could set the management plugin for the proxy apps created on `gs_plugin_eos_create_proxy_app` to `flatpak`, but I am not sure if that will cause other issues.

Reply:
Or just keep this change but not use `goto`, but I think the change is so small/simple that I wouldn't bother.

Reply:
Yep, agreed.